ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/forms/deposit/DepositForm.jsx

```diff
@@ -1,39 +1,369 @@
-  const context = formConfig.createUrl? submitContextType.create : submitContextType.update
+  const context = formConfig.createUrl
+    ? submitContextType.create
+    : submitContextType.update;
   const { onSubmit, onSubmitError } = useOnSubmit({
     apiUrl: formConfig.createUrl || formConfig.updateUrl,
     context: context,
     onSubmitSuccess: (result) => {
       window.location.href = editMode
         ? currentPath.replace("/edit", "")
-        : currentPath.replace("_new", result.id)
-    }
+        : currentPath.replace("_new", result.id);
+    },
   });
-
+  const sidebarRef = useRef(null);
+  const initialValues = {
+    ...record,
+    // additionalTitles: [
+    //   {
+    //     title: {
+    //       cs: "wadawdwadwad",
+    //       ab: "dwadawdadw",
+    //     },
+    //     titleType: "alternative-title",
+    //   },
+    //   {
+    //     title: {
+    //       ab: "dwadadawda",
+    //       am: "dwadawdadwadwa",
+    //     },
+    //     titleType: "subtitle",
+    //   },
+    // ],
+    // abstract: [
+    //   { lang: "cs", value: "ducciano" },
+    //   { lang: "en", value: "Ducciano" },
+    // ],
+    notes: ["dwadwadwad", "dawdadwad", "dadwada"],
+    geoLocations: [
+      {
+        geoLocationPlace: "Ducica's place",
+        geoLocationPoint: {
+          pointLongitude: 130.3232,
+          pointLatitude: 82.44242,
+        },
+      },
+    ],
+  };
+  // fake boolean to simulate if we are editing existing or creating new item
+  const editMode = false;
   return (
     <Container>
       <BaseForm
         onSubmit={onSubmit}
         formik={{
-          initialValues: record,
+          initialValues: initialValues,
           validationSchema: NRDocumentValidationSchema,
           validateOnChange: false,
           validateOnBlur: false,
           enableReinitialize: true,
         }}
       >
-        <pre>Add your deposit form fields here 👇</pre>
+        <Grid>
+          <Grid.Column mobile={16} tablet={16} computer={12}>
+            <Overridable id="NrDocs.Deposit.AccordionFieldFiles.container">
+              <AccordionField
+                includesPaths={["files.enabled"]}
+                active
+                label={i18next.t("Files")}
+              ></AccordionField>
+            </Overridable>
+            <Overridable id="NrDocs.Deposit.AccordionFieldBasicInformation.container">
+              <AccordionField
+                includesPaths={[
+                  "resourceType",
+                  "title",
+                  "additionalTitles",
+                  "publicationDate",
+                  "abstract",
+                  "rights",
+                ]}
+                active
+                label={i18next.t("Basic information")}
+              >
+                <Overridable id="NrDocs.Deposit.PIDField.container"></Overridable>
+
+                <Overridable
+                  id="NrDocs.Deposit.ResourceTypeField.container"
+                  fieldPath="resourceType"
```

Comment:
Beware that for each `fieldPath` prop, everything is nested under `metadata` prefix. So in this case, `fieldPath` will be `metadata.resourceType`. This is different from vocabulary records, which are specific with all metadata not nested under `metadata` key.